### ISSUE TITLE
Telegram Configuration

### ISSUE THREAD

**Issue by erolg** [Huseyinnurbaki/crypto-watchdog#5]
I am trying Telegram integration as follows

```
docker run -d --env TELEGRAM_CHANNEL_HOOK=https://api.telegram.org/botxxxxx:xxxxxxx/sendMessage\?chat_id\=-xxxx --env HOURLY_PERCENTAGE=12 hhaluk/crypto-watchdog
```
It's not working :(.

If I make a request to `/crypto-watchdog/cryptocurrencies` path, I can successfully get statics.

Also if I make a request to telegram API it also works. 
Like that: `https://api.telegram.org/botxxxxx:xxxxxxx/sendMessage\?chat_id\=-xxxx&text=text`

am I missing something?

**Comment by Huseyinnurbaki** [comment]
Hi @erolg 
Thanks for reporting.

Your configuration is correct. Telegram hooks are temporarily disabled with the  current latest version (0.3.3).
"New released tokens" notifications currently not supported with Telegram Hooks.

Next minor version (3.4.3) will include the fixes, which will be released shortly.
Meantime you can use version 0.2.1 to get Telegram notifications.

```
docker run -d --env TELEGRAM_CHANNEL_HOOK=https://api.telegram.org/botxxxxx:xxxxxxx/sendMessage\?chat_id\=-xxxx --env HOURLY_PERCENTAGE=12 hhaluk/crypto-watchdog:0.2.1

```


I will close the issue after I release the fixes.

**Comment by erolg** [comment]
![Screen Shot 2021-05-16 at 15 10 53](https://user-images.githubusercontent.com/5674144/118396605-eaf16d00-b658-11eb-85ca-6ed84c8c966a.png)

Telegram integration works with 0.2.1 \o/

But it seems to have different problems :(

**Comment by Huseyinnurbaki** [comment]
Hi,

Can you try with **&parse_mode=markdown** at the end of your url  ? 

```
https://api.telegram.org/botXXXX:XXXXX/sendMessage?chat_id=XXXXXX&parse_mode=markdown
```

I will also provide a fix & update readme for this one.

**Comment by erolg** [comment]
What do you think about getting only token and chat_id variables from users instead of all webhook URL ?

**Comment by Huseyinnurbaki** [comment]
Telegram api url can include many other queries, taking only the token&chat Id would limit the api. 
Also other webhook urls are way longer, harder to break down, the approach here is consuming webhook url's the same way for all channels.

**Comment by Huseyinnurbaki** [comment]
Hi,

Released version 4.0.0.  I'm closing the issue.
Feel free to reopen if anything goes wrong.
